### ISSUE TITLE
Styling subgraph?

### ISSUE THREAD

**Issue by sinedied** [knsv/mermaid#391]
It would be nice to be allow to style `subgraph`, the same way we can style nodes, for example like this to change subgraph background color:

```
graph TD
  subgraph A
    B-->C
    B-->D
    C-.-D
  end
  D-->E
  classDef red fill:#fdc
  class A red
```

What do you think?

**Comment by knsv** [comment]
I think its a great feature! And it is a great small functionality to start with for someone who want to to try to contribute.

**Comment by WalshJP** [comment]
I'll take this feature!

**Comment by knsv** [comment]
That stunds great. It's yours!

**Comment by yech1990** [comment]
Is that finished yet?

**Comment by tscislo-lingaro** [comment]
Any progress on this?

**Comment by MoeweX** [comment]
Also interested, any progress yet?

**Comment by pchaganti** [comment]
👍

**Comment by joeto0** [comment]
any news?

**Comment by beardedeagle** [comment]
where are we with this?

**Comment by kbsizer** [comment]
This would **_really_** be handy about now.  
Any hope of adding this soon-ish?

**Comment by djoffe** [comment]
@knsv This would be a really handy feature, but it does not look like @WalshJP is aware this is assigned to him...
Maybe remove the `assigned` tag and put it back for grabs?

**Comment by TKappatsch** [comment]
Need it here as well

**Comment by fchiorascu** [comment]
+

**Comment by omerbensa** [comment]
Bump

**Comment by ianqueue** [comment]
Bump

**Comment by Qix-** [comment]
Here's the downside: subgraph titles are also their IDs in OP's case. There would have to be a change to the DSL, something like:

```mermaid
subgraph some-id[Some Subgraph]
...
end
```

**Comment by Qix-** [comment]
Submitted #845, which adds styling. Unfortunately, the maintainer seems to be absent.

**Comment by knsv** [comment]
Wow! A fix for this one, wonderful!

**Comment by fbertoia** [comment]
Why this issue has been closed ? It is not yet fixed, isn't it ?

**Comment by chris579** [comment]
It meant to be closed with #845. If this issue persists for you please create a new one with a reproduction sample.
